sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_operations/_operations.py

```diff
@@ -52,7 +52,7 @@
 _SERIALIZER.client_side_validation = False
 
 
-def build_document_translation__begin_start_translation_request(  # pylint: disable=name-too-long
+def build_document_translation__begin_translation_request(  # pylint: disable=name-too-long
```

Comment:
Sorry missed commenting that we need to update for the renames in the sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_operations/_patch.py + async as well

Reply:
Updated